lab-claudia/app/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <head>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <title> Cowsay Client </title>
+  </head>
+  <body>
+
+    <nav ng-controller="NavController as navCtrl">
+      <i ng-class="moving" class="fa fa-star fa-3x"> </i>
+    </br></br>
+      <button type="button"
+      ng-click="moving='move'"> Move Star</button>
+      <button type="button"
+      ng-click="moving=''"> Move Back</button>
+      <ul>
+        <li ng-repeat="navItem in navCtrl.routes">
+          <a ng-click="navCtrl.selectedItem($active)" ng-href="#">
+            {{navItem.name}} </a>
+        </div>
+        </li>
+      </ul>
+    </nav>
+
+    <section ng-controller="CowsayController as cowsayCtrl"
+    ng-init="cowsayCtrl.currentCow = 'dragon'">
+
+      <!-- directives change functionality of a tag-->
+    <h1> {{cowsayCtrl.title}} </h1>
+    <form ng-submit="cowsayCtrl.helloClick(cowsayCtrl.text)"
+      novalidate
+      name="cowsayForm"
+      ng-class="{'app-red': cowsayForm.$invalid, 'app-green': cowsayForm.$valid}" >
+      <input type="text" name="input" ng-model="cowsayCtrl.text" ng-required="true" ng-minlength="4" ng-maxlength ="10">
+      <select ng-model="cowsayCtrl.currentCow">
+        <option ng-repeat="cowfile in cowsayCtrl.cowfiles" value="{{cowfile}}">
+          {{cowfile}}
+        </option>
+      </select>
+
+      <button type="submit"
+      ng-class="{blue: hover}"
```

Comment:
@loomnugget best to indent these attributes over 1 tab (2 spaces) if you are going to break them onto new lines like this